frontend/src/scenes/dashboard/Dashboard.tsx

```diff
@@ -56,7 +60,7 @@ function DashboardView(): JSX.Element {
     }, [])
 
     useKeyboardHotkeys(
-        dashboardMode === DashboardMode.Public || dashboardMode === DashboardMode.Internal
+        location === DashboardLocation.Public || location === DashboardLocation.InternalMetrics
```

Comment:
Non blocking nit.

The word `location` kind of feels off here. Location isn't really the question whose answer is "public" or "internal metrics", it's more "mode", "state" or "scope"...?

Or what about `DashboardPlacement`?

Reply:
Oooo yeah, I like `placement` way more. Just updated to that.